dm/pkg/checker/table_structure.go

```diff
@@ -456,7 +461,9 @@ func (c *ShardingTablesChecker) checkShardingTable(ctx context.Context, r *Resul
 				r.Extra = fmt.Sprintf("error on sharding %s", c.targetTableID)
 				r.Instruction = "please set same table structure for sharding tables"
```

Comment:
these fields (except Errors) may be overwritten in next loop, at that time output is misleading. Maybe we return on first error is OK.

@GMHDBJD